src/coreclr/src/inc/pedecoder.h

```diff
@@ -402,7 +407,7 @@ class PEDecoder
     IMAGE_COR20_HEADER *FindCorHeader() const;
     CORCOMPILE_HEADER *FindNativeHeader() const;
     READYTORUN_HEADER *FindReadyToRunHeader() const;
-
+    
```

Comment:
Nit: Extra whitespace

Reply:
Thanks for spotting, fixed in 16th commit.